tensorflow/python/ops/math_ops.py

```diff
@@ -3305,6 +3305,14 @@ def log_sigmoid(x, name=None):
   Specifically, `y = log(1 / (1 + exp(-x)))`.  For numerical stability,
   we use `y = -tf.nn.softplus(-x)`.
 
+  Usage Example:
+   >>> tensor1 = [0.0, 1.0, 2.0, 3.0, 4.0, 5.0]
+   >>> tensor2 = [2.0, 5.7]
+   >>> print(tf.math.log_sigmoid(tensor1, name='log_sigmoid_tensor1'))
```

Comment:
You don't need to use print() here.

Reply:
Thanks for the help, my most recent commit should fix these issues.